far/FarRus.hlf.m4

```diff
@@ -3575,62 +3577,64 @@ CD-RW, CD-RW/DVD, DVD-ROM, DVD-RW и DVD-RAM. Функция определен
 разработан для Far 1.7x и не поддерживает всех возможностей текущей версии Far (в частности,
 символов Unicode в именах файлов и в редакторе).
 
-   См. также:
+    См. также:
    
-      Список ~макроклавиш~@KeyMacroDisksList@, доступных в меню выбора диска.
-      Клавиатурные команды, общие ~для всех меню~@MenuCmd@.
+    Список ~макроклавиш~@KeyMacroDisksList@, доступных в меню выбора диска.
+    Клавиатурные команды, общие ~для всех меню~@MenuCmd@.
 
 @ChangeDriveMode
 $ #Настройка меню выбора диска#
-    Диалог позволяют управлять показом информации в ~Меню выбора диска~@DriveDlg@.
+    Диалог позволяет управлять показом информации в
+~меню выбора диска~@DriveDlg@.
 
     #Показывать тип диска#
-    Показать тип диска: "жёсткий", "сетевой" и т.п. 
-    Комбинация клавиш в меню: Ctrl+1
+    Показать тип диска: “жёсткий”, “сетевой” и т.п.
+    Комбинация клавиш в меню: #Ctrl+1#.
 
     #Показывать метку диска#
     Показать метку диска (если есть).
-    Комбинация клавиш в меню: Ctrl+3
+    Комбинация клавиш в меню: #Ctrl+3#.
 
     #Показывать тип файловой системы#
-    Показать тип диска: "NTFS", "FAT" и т.п.
-    Комбинация клавиш в меню: Ctrl+4
+    Показать тип файловой системы: “NTFS”, “FAT” и т.п.
+    Комбинация клавиш в меню: #Ctrl+4#.
 
     #Показывать размер#
-    Показать общее и свободное место на диске.
-    Комбинация клавиш в меню: Ctrl+5
+    Показать размер диска и количество свободного места на нём.
+    Комбинация клавиш в меню: #Ctrl+5#.
 
     #Показывать размер в виде десятичной дроби#
-    Показать общее и свободное место на диске в виде десятичной дроби.
-    Комбинация клавиш в меню: Ctrl+5 (повторное нажатие).
+    Показать размер диска и количество свободного места в виде
+десятичной дроби.
```

Comment:
I am not sure this describes the actual behavior. It looks like Far switches between two measurement units (M vs. G or G vs. T, etc.) Should I describe the actual behavior (and adjust `Change Drive Menu Options` dialog accordingly) or would you suggest modifying how `Change drive` menu shows sizes?

Reply:
If checked it's rational (1.04 G), otherwise integer (1,061 M).
The behaviour is the same as in file panels and probably already described there.

Reply:
Do you mean the `C` modifier to file size column type (`S`)? Then, the description is not very helpful:
```
C - format file size;
```

And the behavior seems to be somewhat different.

Reply:
Not helpful indeed. Would you like to improve it? :)

Reply:
Yeah! I'll add some explanations with the next PR. I'll also try to come up with a better wording for `Ctrl+5` in `Change drive` menu.